PMMain/Midas/MidasGUI/DistanceWidget.h

```diff
@@ -0,0 +1,43 @@
+#ifndef DISTANCE_WIDGET_H
+#define DISTANCE_WIDGET_H
+
+#include "DistanceBar.h"
+#include "MidasCommon.h"
+#include "MidasThread.h"
+#include "qlabel.h"
+
+
+class DistanceWidget : public QWidget
+{
+    Q_OBJECT
+public:
+    DistanceWidget(MidasThread *mainThread, QWidget *parent = 0,
+        int width = DISTANCE_DISPLAY_WIDTH, 
+        int height = MOUSE_INDICATOR_SIZE);
```

Comment:
I removed the defaults. Anyone using it will just have to set the height and width.